content/rs/administering/_index.md

```diff
@@ -5,10 +5,24 @@ weight: 50
 alwaysopen: false
 categories: ["RS"]
 ---
-This section covers everything you need to know to get up and running
-with RS, included installing, configuring a cluster, maintaining a
-cluster, creating databases, etc.
+This section covers everything you need to know to get up and running with RS.
 
-Topics include:
+{{% note %}}
+In addition to using the RS web UI to manage the cluster,
+you can also access cluster operations through the cluster REST API.
+
+To access the cluster REST API documentation, either:
+
+- Open the HTML files in the RS software package:
+    1. Download the RS software package.
+    1. Unarchive the package.
```

Comment:
I think we need to write here- install the RS.... (instead of the second point)

Reply:
I noticed now that you mention it later. Is the first "option" approved by the Cluster team and working?